ncm-sudo/src/main/perl/sudo.pm

```diff
@@ -197,8 +199,11 @@ sub generate_privilege_lines {
         my $ln = $info{PRIVILEGE_USER()}->getValue;
         $ln .= "\t" . $info{PRIVILEGE_HOST()}->getValue;
         $ln .= "= (". $info{PRIVILEGE_RUNAS()}->getValue . ")\t";
-        $ln .= $info{PRIVILEGE_OPTS()}->getValue . "\t"
-        if defined $info{PRIVILEGE_OPTS()};
+	if (defined($info{PRIVILEGE_OPTS()})) {
+		$ln .= $info{PRIVILEGE_OPTS()}->getValue;
+		$ln .= ':' if $ln !~ /:$/;
```

Comment:
sounds like some backwards incompatible commits were added, and lack of unittests didn't catch this change.
if you can, you may always add a unittest for this behaviour (ensure the trailing `:` in the privilige opts). if not, open an issue to add this unittest.